provider/source/src/tests/make_testdata.rs

```diff
@@ -76,6 +76,7 @@ fn make_testdata() {
             "meter" | "foot" | "kilogram" | "pound" | "hour" | "minute" | "second"
         )
     })
+    .with_marker_attributes_filter("currency", |attrs| matches!(attrs.as_str(), "CAD" | "EUR" | "USD"))
```

Comment:
Suggestion (optional): probably restore the previous list `"USD", "CAD", "EUR", "GBP", "EGP"` to make this a more minimal change. I don't remember exactly why we chose that set, but I think "EGP" at least was interesting because of plural forms or something.